src/components/TeamGrid/index.scss

```diff
@@ -0,0 +1,106 @@
+@use "/src/stylesheets/fonts";
+@use "/src/stylesheets/responsiveness";
+@import "../../stylesheets/variables";
+@import "../../stylesheets/fonts";
+
+.Member {
+  animation: fade-in 1s;
+  width: 90%;
+  max-width: 292px;
+  min-height: 392px;
+  margin: 30px;
+
+  @media (max-width: 320px) {
+    margin: 250px;
+  }
+  border-radius: 27px;
+  border: 4px solid transparent;
+
+  &--Image {
+    display: block;
+    margin-left: auto;
+    margin-top: 10%;
+    margin-right: auto;
+    height: 250px;
+    width: 250px;
```

Comment:
width: 10vw; possibly ?

Reply:
Also, you could change the flexbox so it goes to column (puts the logo above the text) when the screen size is mobile.